auction-app-backend/src/main/resources/application.yml

```diff
@@ -1,21 +1,21 @@
 spring:
   datasource:
-    url: jdbc:postgresql://dpg-cp0h51q1hbls73eaktmg-a.frankfurt-postgres.render.com/auction_app_cqgi
```

Comment:
Why are we constantly changing this configuration?

Reply:
since the database deployed had a few migrations from the previous ticket that were not merged yet, instead of managing those migrations manually i decided to switch to use my local db without those new migrations until they are approved of